src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java

```diff
@@ -67,11 +67,16 @@ public void actionPerformed(@NotNull AnActionEvent e) {
                 return;
             }
         }
-        if (buildFile == null) {
+
+        boolean isActionFromShiftShift = "GoToAction".equalsIgnoreCase(e.getPlace());
+
+        if (isActionFromShiftShift || buildFile == null) {
```

Comment:
If `isActionFromShiftShift` is true, do we need to assign a value to `buildFile` here, or is that only in the case where `buildFile` is null? It seems to me that we only care about assigning the `buildFile` at this point if `buildFile` was null to start (in other words, `isActionFromShiftShift` doesn't seem to have an affect on whether or not we should assign a value to the `buildFile`.)

If my assumption is correct, I would recommend taking out the `isActionFromShiftShift` boolean from the first `if` statement:
```
if (buildFile == null) {
    // set build file
}

if (isActionFromShiftShift || buildFile == null) {
    // prompt user for selection
}
 ```

Reply:
I just did the adjustment for it. Looking at GitHub changes can give the impression that it has more changes now but it has not. It is just that the if's are not nested anymore.